src/prachi/Assignment_17/ArrNumberFrequency.java

```diff
@@ -0,0 +1,29 @@
+package prachi.Assignment_17;
+
+import java.util.Scanner;
+
+/*Program 4:  From given numbers, return the frequency of target number.
+input : 10,23,23,44,23,10,23,4,23
+target number : 23         
+output : Freq of 23 is 5*/
+
+public class ArrNumberFrequency {
+
+	int getNumFreq(int[] numberArr, int target) {
+		int targetCnt = 0;
+		for (int index = 0; index < numberArr.length; index++) {
+			if (target == numberArr[index])
+				targetCnt++;
```

Comment:
Here,You should return true or false, as we have to check that number is present in array or not.
Using this targetCnt++ You are not checking that number is present in array.

Hope you got my point